VaticanBAV/et/BAVet120.xml

```diff
@@ -28,148 +28,247 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<item xml:id="q4" n="3"><num value="8">፰</num><dim unit="leaf">10</dim><locus from="23r" to="32v"></locus></item>
-<item xml:id="q5" n="4"><num value="9">፱</num><dim unit="leaf">10</dim><locus from="33r" to="42v"></locus></item>
-<item xml:id="q6" n="5"><num value="1">፩</num><dim unit="leaf">10</dim><locus from="43r" to="52v"></locus></item>
-<item xml:id="q7" n="6"><num value="2">፪</num><dim unit="leaf">10</dim><locus from="53r" to="62v"></locus></item>
-<item xml:id="q8" n="7"><num value="3">፫</num><dim unit="leaf">10</dim><locus from="63r" to="72v"></locus></item>
-<item xml:id="q9" n="8"><num value="4">፬</num><dim unit="leaf">10</dim><locus from="73r" to="82v"></locus></item>
-<item xml:id="q10" n="9"><num value="5">፭</num><dim unit="leaf">10</dim><locus from="83r" to="92v"></locus></item>
-<note>The quires are numbered. Quires no. 6-9 (ff. 3-42) contain the text of
-  Śәna fәṭrat, quires no. 1-5 (ff. 43-92) contain ʾAmmǝstu ʾaʿǝmāda mǝśṭir and
-  blank pages</note>
-</list>
-</collation>
-</supportDesc>
-<layoutDesc><layout writtenLines="24"></layout></layoutDesc>
-</objectDesc>
-
-<handDesc>
-<handNote xml:id="h1" script="Ethiopic">
-<seg type="ink"> Black, red</seg>
-<seg type="rubrication">Titles</seg>
-<date when="1923"></date>
-</handNote>
-</handDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc> 
+                                <support>
+                                    <material key="parchment">                                
+                            </material>
+                            </support>
+                                <extent>
+                                    <measure unit="leaf">92</measure>
+                                    <measure unit="leaf" type="blank">
+                                        <locus from="90r" to="92v"/>
+                                    </measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>224</height>
+                                        <width>166</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <note>The quires are numbered with Ethiopic numerals. Quire marks are written in the main hand in red ink on the upper inner corner of the recto of the first folio of each quire.
+                                        Quires 1–5 (<locus from="3r" to="42v"/>) are numbered 6 to 9; quires 6–9 (<locus from="43r" to="92v"/>) are numbered 1 to 5.
+                                        According to <bibl>
+                                            <ptr target="bm:GrebTiss1935Codices"/>
+                                            <citedRange unit="page">488</citedRange>
+                                        </bibl>, this would indicate that the quires were bound in the reverse order of what was originally planned.
```

Comment:
It would mean, that the second part of the codex was originally the first part. If the quires were bound 'in the reverse order', as you say, the order would be: 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, but not 6, 7, 8, 9, 1, 2, 3, 4, 5 as you wrote above.

Reply:
The intended meaning is that the two _halves_ of the textblock were bound in reverse order. I have corrected the sentence accordingly.